ci/do_ci.sh

```diff
@@ -150,7 +150,7 @@ if [[ $# -ge 1 ]]; then
 else
   # Coverage test will add QUICHE tests by itself.
   COVERAGE_TEST_TARGETS=("//test/...")
-  TEST_TARGETS=("${COVERAGE_TEST_TARGETS[@]}" "@com_googlesource_quiche//:ci_tests")
+  TEST_TARGETS=("${COVERAGE_TEST_TARGETS[@]}" "//tools/..." "@com_googlesource_quiche//:ci_tests")
```

Comment:
It might be sufficient to just put `bazel build "${BAZEL_BUILD_OPTIONS[@]}" -c dbg //tools/...` as an explicit step in `bazel.debug`.

Reply:
There are tests under tools that are not running.  It is not just about build.